client/src/pages/resources/index.tsx

```diff
@@ -115,6 +117,18 @@ const Resources = (): ReactElement => {
               <Radio checked={edit} slider onClick={() => setEdit(!edit)} />
             </AdminView>
           </div>
+          {!edit && (
+            <Button
+              className="calendly-button"
+              onClick={() =>
+                openPopupWidget({
+                  url: 'https://calendly.com/sawhney4/60min',
+                })
+              }
+            >
+              Schedule Office Hours
+            </Button>
+          )}
```

Comment:
This button style seems to be something that Bryan uses throughout the designs a few times, it would be cool if we could turn it into a component!